apis/server/image_bridge_test.go

```diff
@@ -36,3 +42,81 @@ func Test_pullImage_without_tag(t *testing.T) {
 	}
 	s.pullImage(context.Background(), nil, req)
 }
+
+func Test_pullImage_counter(t *testing.T) {
+	var s Server
+	ctx := context.Background()
+
+	key := fmt.Sprintf(`engine_daemon_image_actions_counter_total{action="%s"}`, "pull")
+	keySuccess := fmt.Sprintf(`engine_daemon_image_success_actions_counter_total{action="%s"}`, "pull")
+	countBefore, countSuccessBefore := getMetric(ctx, t, &s, key, keySuccess)
+
+	ch := make(chan int, 1)
+	go func() {
+		s.ImageMgr = &mockImgePull{
+			ImageMgr: &mgr.ImageManager{},
+			handler: func(ctx context.Context, imageRef string, authConfig *types.AuthConfig, out io.Writer) error {
+				assert.Equal(t, "reg.abc.com/base/os:7.2", imageRef)
+				time.Sleep(1 * time.Minute)
```

Comment:
I think we don't need to wait for one minute. When we get the `countBefore` value, then we can start pull image. `1m` is too long for us. we need to run test case fast.

Reply:
Agree, one minute is too long if it is not necessary.

Reply:
Done. I modified it to 2s.